RuriLib/Blocks/Interop/Methods.cs

```diff
@@ -40,14 +41,57 @@ public static string ShellCommand(BotData data, string executable, string argume
          * of writing C# code that calls these methods where necessary once it's transpiled.
          */
 
-        public static async Task<T> InvokeNode<T>(BotData data, string scriptFile, object[] parameters)
+        public static async Task<T> InvokeNode<T>(BotData data, (string, string) scriptOrFile, object[] parameters, bool isScript = false)
```

Comment:
This is a breaking change to the signature of the function, old configs that have been compiled with an old build will not work anymore. A valid signature should be

```cs
public static async Task<T> InvokeNode<T>(BotData data, string scriptOrFile, object[] parameters, bool isScript = false, string scriptHash = null)
```